Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,184 @@
+          pradeo.initiator.emm: '{{json_event.message.device.emmDeviceInfo.emm}}'
+      - filter: '{{json_event.message.source == "admin" and json_event.message.user != null}}'
+        set:
+          pradeo.initiator.id: '{{json_event.message.user.id}}'
+          pradeo.initiator.email: '{{json_event.message.user.email}}'
+          pradeo.initiator.firstName: '{{json_event.message.user.firstName}}'
+          pradeo.initiator.lastName: '{{json_event.message.user.lastName}}'
+  permission:
+    actions:
+      - filter: '{{json_event.message.type == "DevicePermissionStatusUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.device.email}}'
+          pradeo.device.imei: '{{json_event.message.content.device.imei}}'
+          pradeo.device.byod: '{{json_event.message.content.device.byod}}'
+          pradeo.device.declaredOperatingSystem: '{{json_event.message.content.device.declaredOperatingSystem}}'
+          pradeo.device.declaredOperatingSystemVersion: '{{json_event.message.content.device.declaredOperatingSystemVersion}}'
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: '{{json_event.message.content.device.declaredOperatingSystemSecurityPatchDate}}'
+          pradeo.device.declaredModel: '{{json_event.message.content.device.declaredModel}}'
+          pradeo.device.lastConnection: '{{json_event.message.content.device.enrollmentStatus.lastConnection}}'
+          pradeo.device.coupled: '{{json_event.message.content.device.enrollmentStatus.coupled}}'
+          pradeo.device.mdmId: '{{json_event.message.content.device.emmDeviceInfo.externalId}}'
+          pradeo.device.emm: '{{json_event.message.content.device.emmDeviceInfo.emm}}'
+          pradeo.permission.name: '{{json_event.message.content.permission}}'
+          pradeo.permission.value: '{{json_event.message.content.value}}'
+  detection:
+    actions:
+      - filter: '{{json_event.message.type == "DeviceDetection"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.id}}'
+          pradeo.detection.status: '{{json_event.message.content.metric}}'
+          pradeo.detection.value: '{{json_event.message.content.status}}'
+      - filter: '{{json_event.message.type == "DeviceSystemStatusRecordUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceSystemStatusRecord.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceSystemStatusRecord.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceSystemStatusRecord.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceSystemStatusRecord.device.email}}'
```

[`user.email`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email) is more suitable.
```suggestion
          user.email: '{{json_event.content.deviceSystemStatusRecord.device.email}}'
```